docker/consul-template-bookbrainz.conf

```diff
@@ -0,0 +1,10 @@
+template {
+    source = "/home/bookbrainz/bookbrainz-site/config.json.ctmpl"
+    destination = "/home/bookbrainz/bookbrainz-site/config/config.json"
+}
+
+exec {
+    command = "/usr/local/bin/bookbrainz"
```

Comment:
It'd be good to run node as non-root, e.g. the bookbrainz user, if you can. The `chpst` command is an easy way to drop privileges. Though you might have to adjust the permissions of the generated `config.json` after it's written too (since consul-template runs & writes stuff as root). See https://github.com/metabrainz/coverart_redirect/blob/master/docker/prod/consul-template-redirect.conf#L4-L8